src/Tools/dotnet-dump/Commands/SOSCommand.cs

```diff
@@ -0,0 +1,90 @@
+    [CommandAlias(Name = "dumpobj",  Expansion = "sos DumpObj", Help = "Displays info about an object at the specified address.")]
+    [CommandAlias(Name = "dumpstack",  Expansion = "sos DumpStack", Help = "Displays a native and managed stack trace.")]
+    [CommandAlias(Name = "dso",  Expansion = "sos DumpStackObjects", Help = "Displays all managed objects found within the bounds of the current stack.")]
+    [CommandAlias(Name = "eeheap",  Expansion = "sos EEHeap", Help = "Displays info about process memory consumed by internal runtime data structures.")]
+    [CommandAlias(Name = "eestack",  Expansion = "sos EEStack", Help = "Runs dumpstack on all threads in the process.")]
+    [CommandAlias(Name = "finalizequeue",  Expansion = "sos FinalizeQueue", Help = "Displays all objects registered for finalization.")]
+    [CommandAlias(Name = "gcroot",  Expansion = "sos GCRoot", Help = "Displays info about references (or roots) to an object at the specified address.")]
+    [CommandAlias(Name = "ip2md",  Expansion = "sos IP2MD", Help = "Displays the MethodDesc structure at the specified address in code that has been JIT-compiled.")]
+    [CommandAlias(Name = "loadsymbols",  Expansion = "sos SetSymbolServer -loadsymbols", Help = "Load the .NET Core native module symbols.")]
+    [CommandAlias(Name = "name2ee",  Expansion = "sos Name2EE", Help = "Displays the MethodTable structure and EEClass structure for the specified type or method in the specified module.")]
+    [CommandAlias(Name = "pe",  Expansion = "sos PrintException", Help = "Displays and formats fields of any object derived from the Exception class at the specified address.")]
+    [CommandAlias(Name = "syncblk",  Expansion = "sos SyncBlk", Help = "Displays the SyncBlock holder info.")]
+    [CommandAlias(Name = "histclear",  Expansion = "sos HistClear", Help = "Releases any resources used by the family of Hist commands.")]
+    [CommandAlias(Name = "histinit",  Expansion = "sos HistInit", Help = "Initializes the SOS structures from the stress log saved in the debuggee.")]
+    [CommandAlias(Name = "histobj",  Expansion = "sos HistObj", Help = "Examines all stress log relocation records and displays the chain of garbage collection relocations that may have led to the address passed in as an argument.")]
+    [CommandAlias(Name = "histobjfind",  Expansion = "sos HistObjFind", Help = "Displays all the log entries that reference an object at the specified address.")]
+    [CommandAlias(Name = "histroot",  Expansion = "sos HistRoot", Help = "Displays information related to both promotions and relocations of the specified root.")]
+    [CommandAlias(Name = "setsymbolserver",  Expansion = "sos SetSymbolServer", Help = "Enables the symbol server support ")]
+    [CommandAlias(Name = "soshelp",  Expansion = "sos Help", Help = "Displays all available commands when no parameter is specified, or displays detailed help information about the specified command. soshelp <command>")]
```

Comment:
We should be careful with this one so it doesn't print info about commands we don't support. That might require some factoring inside SOS?

Reply:
"soshelp" is the same help that is used from lldb so refactoring the native SOS is complicated. 

In a future PR, I was going to try to plumb together the "help <sos-alias>" command to also display the soshelp from the help text file.  It would still require filtering out the unsupported commands.

I'd rather do all of the "filtering" in the SOSHost/SOSCommand and not in native SOS. Hopefully you would be ok putting this off until a future PR.

Reply:
yep, totally fine to put that in a later PR. Lets open an issue to track dealing with it though.